ci/benchmark/build.sh

```diff
@@ -39,6 +39,9 @@ export LIBCUDF_KERNEL_CACHE_PATH="$HOME/.jitify-cache"
 # Dask & Distributed option to install main(nightly) or `conda-forge` packages.
 export INSTALL_DASK_MAIN=1
 
+# Dask version to install when `INSTALL_DASK_MAIN=0`
+export DASK_STABLE_VERSION="2022.8.0"
```

Comment:
Want to note that the relevant fixes for the mixed stable/nightly issue here are in https://github.com/conda-forge/dask-feedstock/pull/191 and https://github.com/conda-forge/distributed-feedstock/pull/218, since our problem is that when installing conda-forge `dask` we ended up pulling in nightly `dask-core`/`distributed`.

With this context, https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/312 applies this change across all stable `dask`/`distributed` packages that would've had this issue, so once that's in we shouldn't have to bump the stable version here

Reply:
Awesome, I'll hold off until https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/312 is merged and will revert back to the prev stable version that we were pointing to as a minimum.